phpspec.yml.ci

```diff
@@ -0,0 +1,10 @@
+suites:
+  data_import_suite:
+    namespace: Ddeboer\DataImport
+    psr4_prefix: Ddeboer\DataImport
+extensions:
+  - PhpSpec\Extension\CodeCoverageExtension
+formatter.name: pretty
+code_coverage:
```

Comment:
>> they should make sense to developers

> I am not sure what that means, but

I mean: PHPUnit tests provide pieces of code that app developers using our library can just use (copy/paste) into their projects: they see the tests and they immediately find out how to use the library. This is different with Behat, where you have to switch between the Gherkin stories and the underlying step definitions in PHP.